docs/core-concepts/authenticating-with-api-keys.mdx

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Authenticating with API Keys'
+---
+
+# Authenticating with API Keys
+
+The Regression Games platform provides the ability to generate API keys, which can be used
+to authenticate against our API. This is the recommended way to authenticate against our API
+when connecting external systems to Regression Games.
+
+You can find the **API Keys** manager at the bottom of your [Regression Games account settings](https://play.regression.gg/account).
+
+## Generating an API Key
+
+Enter a unique name for your key and then click **Generate Key**.
+API keys have a default expiration date of 1 year.
+
+:::danger
+Make sure to copy the generated key and store it somewhere safe.
+Once the success dialog is dismissed, you will never be able to see the generated value through the Regression Games client again!
+:::
+
+
+![Create an API Key](img/api-keys/create-new-api-key.gif)
+
+
+## Using the API Key
+
+Once you have generated an API key, you can use it to authenticate against our API.
+
+### API Requests
+
+Include the key in the `Authorization` header of your request.
```

Comment:
I believe in the flow we are pushing the user will:
> Go to Installing Regression Games page
> Get to Set an API Key
> Click on 'Learn more about our API keys, see the Authentica ...` 
> Come here.

Which means that this information doesn't have a lot of context, how do I send a request? What is a request? Consider moving this down and/or adding a link to the relevant page explaining the request.

Reply:
I can think of two options:
1. Remove the link to this page from the installation guide and assume that if they want to know more then they'll find the api-keys doc themselves. I don't like this because we don't have any search functionality on the site and security is something Zack and Aaron have mentioned as factors that could scare off potential users if not handled correctly.
2. Move the "request" section below the "unity" section so that the order is more relevant to users coming to this page from the installation guide.

> and/or adding a link to the relevant page explaining the request.
This part I disagree with. If a developer doesn't already know what an API request is or how headers work, then this section either isn't relevant to them or they need to learn more about REST APIs in general and it's not our job to explain teach that to them.

Reply:
The point isn't necessarily to teach people about the REST API's in general, but how requests are used in the context of our tool. When we have a page telling people about that we can link it there